### PR TITLE
CON-1956-Content changes to recruit dashboard

### PR DESCRIPTION
**Pull request by ben1stone**
updates recruit dashboard content to use advert instead of vacancy

### CODE REVIEW

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 1]
```diff
@@ -61,7 +61,7 @@
                         </a>
                         <span asp-hide="@Model.HasDraftVacancy">@FilteringOptions.Draft @Model.VacancyTextDraft</span>
                     </h3>
-                    <p class="govuk-body das-card--description">Vacancies that need completing and sending for review.</p>
+                    <p class="govuk-body das-card--description">Adverts that you need tocomplete and send for review.</p>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml:64]: to complete. Need a space.

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 2]
```diff
@@ -151,20 +151,20 @@
                         </a>
                         <span asp-hide="@Model.HasClosedVacancy">@FilteringOptions.Closed @Model.VacancyTextClosed</span>
                     </h3>
-                    <span class="govuk-body das-card--description">Vacancies that have passed the closing date. You can clone these vacancies to republish them.</span>
+                    <span class="govuk-body das-card--description">Adverts that have passed theclosing date. You can clonethese adverts to republish them.</span>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml:154]: the closing

Comment (shomavg) [reply src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml:154]: clone these

File: src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml [hunk 3]
```diff
@@ -105,7 +105,7 @@
                         </a>
                         <span asp-hide="@Model.HasLiveVacancy">@FilteringOptions.Live @Model.VacancyTextLive</span>
                     </h3>
-                    <span class="govuk-body das-card--description">Vacancies on Find an apprenticeship service.</span>
+                    <span class="govuk-body das-card--description">Your adverts on the Find anapprenticeship service.</span>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml:108]: Should there be a space between anapprenticeship?

Comment (ben1stone) [reply src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml]: yes, I've updated

**Pull request merged**